### PR TITLE
[bellbot_voice] Adding ability to incorporate text published on topics.

### PR DESCRIPTION
**Pull request by cdondrup**
This adds the functionality that text from topics publishing string objects can be integrated into the text output, e.g.:

"I am at $(topic /current_node)" would be replaced to "I am at Reception". The publishing topic should be latched because this call is blocking.

This feature allows to publish the description on how to get to a station from the lift and incorporate it into the text output, e.g. "$(topic /my_description). Thank you for using the bellbot." Would translate to whatever is publish on the description topic plus the added string. See config file for usage example.

### CODE REVIEW

**Comment by marc-hanheide** [comment]
kewl!

File: bellbot_action_server/conf/bellbot_voice.yaml [hunk 1]
```diff
@@ -13,8 +13,8 @@ WaitingForGoal:
     start: "Waiting For Goal start"
     end: "Waiting For Goal end"
 Guiding:
-    start: "Guiding start"
-    end: "Guiding end"
+    start: "Guiding start to $(topic /current_node)"
```
Comment (marc-hanheide) [inline bellbot_action_server/conf/bellbot_voice.yaml:16]: This doesn't make much sense... `"Guiding start to $(topic /current_node)"`... but i get the concept.

Comment (cdondrup) [inline bellbot_action_server/conf/bellbot_voice.yaml:16]: I know. This is just a proof of concept. The real config file is in aaf deployment anyway.

**Comment by marc-hanheide** [comment]
Dies this work with the topics we have in bellbot already? I know it works with `/current_node`, but how do you get the destination?

**Comment by cdondrup** [comment]
It works with everything that publishes a string. If this is not published yet, we could add it or I adjust the type to something different. But basically, bellbot has to publish something if this feature should be used.

**Comment by marc-hanheide** [comment]
Why not using the smach `input_keys` and `output_keys`? Aren't they published anyway? In bellbot they contain the target...

**Comment by cdondrup** [comment]
I'll have look, good idea. But that won't work for the description.

**Comment by cdondrup** [comment]
The whole thing will be tricky due to cryptic waypoint names. I would need a more human readable string for the target.

**Comment by marc-hanheide** [comment]
I could publish the "speaking name" from the GUI... I'm working on that now anyway

**Comment by Jailander** [comment]
We could also read the name on the content I can make a service that returns the name from the content

**Comment by marc-hanheide** [comment]
then it's easier I simply publish the content as I already have it...

**Comment by Jailander** [comment]
Yes I  think it is

**Pull request merged**